backends/tc/ebpfCodeGen.cpp

```diff
@@ -1349,44 +1347,43 @@ void ControlBodyTranslatorPNA::processApply(const P4::ApplyMethod *method) {
             bool memcpy = false;
```

Comment:
Can we change this variable name to something like "noEndiannessConversion". This conflicts with libfunc name and is confusing